barter-data/Cargo.toml

```diff
@@ -17,9 +17,9 @@ rust_decimal_macros = { workspace = true }
 
 [dependencies]
 # Barter Ecosystem
-barter-integration = { path = "../barter-integration", version = "0.7.4" }
-barter-instrument = { path = "../barter-instrument", version = "0.1.0" }
-barter-macro =  { path = "../barter-macro", version = "0.1.1" }
```

Comment:
I can't remember why I have the version and path here, but if my memory is correct it's important upon `cargo release`

Reply:
What is the purpose of this commit?

Reply:
I wanted to unify the local crates specifications. I've also added versions. This should be better, because there is a single place where we need to change the versions.

Reply:
That looks better!

Please could you rebase (ie/ `git rebase -i HEAD~4`) in order to: 
1. squash "chore: add versions to local crates" into "chore: source local crates from workspace" 
2. Add semantic commit style to "bump tokio-tungstenite to 0.26" -> "chore: ...".